lib/components/form/user-settings.js

```diff
@@ -42,7 +42,26 @@ function findLocationType(
     ? getFormattedPlaces(match.type, intl)
     : null
 }
-function summarizeQuery(query, intl, locations = []) {
+export function summarizeQuery(query, intl, locations = []) {
+  if (!query.from.name) {
+    query.from.name = intl.formatMessage(
+      { id: 'common.coordinates' },
+      {
+        lat: query.from.lat.toFixed(5),
+        lon: query.from.lon.toFixed(5)
+      }
+    )
```

Comment:
Extract a function to render coordinates, or better, the coordinates of a place (will be used multiple times).

Reply:
For rendering, I suspect we need to use `<FormattedNumber>`.

Reply:
Good call. Refactored in 6d5bc90e